client/my-sites/posts/post-share.jsx

```diff
@@ -212,6 +226,7 @@ export default connect(
 		const userId = getCurrentUserId( state );
 
 		return {
+			planHasRepublicizeFeature: hasFeature( state, siteId, FEATURE_REPUBLICIZE ),
```

Comment:
Since:
1. This is a new feature, so legacy addon concern is beside the point
2. This is a feature only for calypso
3. The list of features in the API is outdated since calypso uses primarily plans/constants
4. We will be totally changing plans distribution when making new version of this feature

I propose to mergi it as it is and have a proper feature gate when we develop a full-fledged version of this feature